portfolio/src/main/webapp/index.html

```diff
@@ -70,8 +69,8 @@ <h3 class="section-subheading text-muted">Our purpose.</h3>
             </span>
             <h4 class="my-3">To Inform</h4>
             <p class="text-muted">
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore
-              harum ex magni, dicta impedit.
+              The goal of Silence Is Violence is to inform the general public. We want to make any and all information
+              regarding police brutality, racial equity, and protests in your area, is at your fingertips.
```

Comment:
Since we're not adhering to one certain area, I think the language should be more general. Our focus is to spread accurate information pertaining to the black community, allies, and anyone who is willing to educate themselves.